### PR TITLE
Fix Bug 1180891, switch to new style newsletter subscription form fixes l10n problems

### PR DESCRIPTION
**Pull request by schalkneethling**

### CODE REVIEW

File: media/css/plugincheck/plugincheck.less [hunk 1]
```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
 #sec-plugin-uptodate {
     display: none;
 }
-.footer-newsletter-form {
-    padding: 40px 20px;
+#mozorg-newsletter-form,
+#newsletter-form {
+    width: 1300px / 2;
```
Comment (craigcook) [inline media/css/plugincheck/plugincheck.less]: This width is making it overflow in narrower windows. I think you don't need a width here at all and could just let it default to auto.
plugincheck-overflow

File: media/css/plugincheck/plugincheck.less [hunk 2]
```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
...
+    border-bottom: none;
+    box-shadow: none;
+
+    h3 {
+        display: block;
+        float: none;
+        width: auto;
+        padding: (@baseLine * 2) 0;
+        color: @textColorSecondary;
+        .font-size(46px);
```
Comment (craigcook) [inline media/css/plugincheck/plugincheck.less]: This seems really big for an h3, even when centered as the main headline (see previous screenshot). I'd probably take this down to 32px.

File: media/css/plugincheck/plugincheck.less [hunk 3]
```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
 #sec-plugin-uptodate {
     display: none;
 }
-.footer-newsletter-form {
-    padding: 40px 20px;
+#mozorg-newsletter-form,
+#newsletter-form {
+    width: 1300px / 2;
+    margin: 0 auto;
+    padding: 0 0 (@baseLine * 2) 0;
```
Comment (craigcook) [inline media/css/plugincheck/plugincheck.less]: Could use some padding on the sides as well (see previous screenshot).

File: media/css/plugincheck/plugincheck.less [hunk 4]
```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
...
+           font-weight: bold;
+           opacity: 1;
+        }
+    }
+
+    .form-submit {
+        width: 50%;
+        margin: 0;
+
+        input[type="submit"] {
```
Comment (craigcook) [inline media/css/plugincheck/plugincheck.less]: Should add `text-align: center;` for the button
plugincheck-button

**Comment by schalkneethling** [comment]
thx @craigcook on it

**Comment by schalkneethling** [comment]
@alexgibson r?

**Comment by alexgibson** [comment]
Issue with the styling still?

newsletter

File: media/css/plugincheck/plugincheck.less [hunk 5]
```diff
@@ -263,8 +263,121 @@ html[dir="rtl"] {
...
+            width: 100%;
+        }
+
+        input[type="email"] {
+            height: 32px;
+            padding: 5px 10px;
+            background: #fbf9f4;
+            border: 2px solid #e1ddd3;
+            border-radius: 7px;
+            -webkit-appearance: none;
```
Comment (alexgibson) [inline media/css/plugincheck/plugincheck.less]: Does this also need `moz-appearance`, or is there a reason only webkit needs this?

File: media/css/plugincheck/plugincheck.less [hunk 6]
```diff
@@ -263,8 +263,121 @@ html[dir="rtl"] {
...
+    text-align: center;
+
+    h3 {
+        display: block;
+        float: none;
+        color: @textColorSecondary;
+        margin: 0;
+        padding: (@baseLine * 2) 0;
+        width: auto;
+        .font-size(2.4rem);
```
Comment (alexgibson) [inline media/css/plugincheck/plugincheck.less]: Nit: use `px` for consistency with the rest of the file

File: media/css/plugincheck/plugincheck.less [hunk 7]
```diff
@@ -263,8 +263,121 @@ html[dir="rtl"] {
...
+            min-height: 45px;
+            background: #0095dd;
+            .font-size(18px);
+            font-weight: normal;
+            text-shadow: none;
+            text-transform: uppercase;
+            text-align: center;
+            box-shadow: none;
+            border-radius: 7px;
+            -webkit-appearance: none;
```
Comment (alexgibson) [inline media/css/plugincheck/plugincheck.less]: Does this need `-moz-appearance` too?

File: media/css/plugincheck/plugincheck.less [hunk 8]
```diff
@@ -263,8 +263,121 @@ html[dir="rtl"] {
...
+        }
+    }
+
+    .privacy-check-label .title {
+        padding: 0 20px 0 25px;
+    }
+}
+.newsletter-container {
+    background-color: #fff;
+    .border-radius(8px);
```
Comment (alexgibson) [inline media/css/plugincheck/plugincheck.less]: No need to use border radius mixin any more

**Comment by alexgibson** [comment]
Thank you message needs responsive styles

thankyou

File: media/css/plugincheck/plugincheck.less (hunk 5 above)
Comment (schalkneethling) [inline media/css/plugincheck/plugincheck.less]: Copied from elsewhere in bedrock but yeah, let me add that. There is no reason I am aware of not to.

File: media/css/plugincheck/plugincheck.less (hunk 8 above)
Comment (schalkneethling) [inline media/css/plugincheck/plugincheck.less]: Indeed. Should we also remove it from `lib.less` then? A little less code

Comment (schalkneethling) [inline media/css/plugincheck/plugincheck.less]: Not in this bug of course, but in a separate bug/pr

**Comment by schalkneethling** [comment]
@alexgibson ok, addressed all the issues mentioned here. The only thing is https://github.com/mozilla/bedrock/pull/3445#issuecomment-155458330

I could no replicate this with this form nor the non en-US one. Checked in Fx, Chrome as well as IE8 and 9

**Comment by schalkneethling** [comment]
@alexgibson Ok, merged with master and fixed the form layout accordingly.

**Comment by alexgibson** [comment]
Looks good, r+ - care to squash?

**Pull request merged**